src/odb/src/db/dbProperty.cpp

```diff
@@ -135,7 +135,9 @@ dbOStream& operator<<(dbOStream& stream, const _dbProperty& prop)
 
   switch (prop._flags._type) {
     case DB_BOOL_PROP:
-      stream << std::get<uint>(prop._value);
+      // Older versions of the spec treated bools as uints
+      // retain backwards compatability
+      stream << static_cast<uint>(std::get<bool>(prop._value));
```

Comment:
You could make this schema version dependent for a bit of improved compactness.  Probably not worth the bother though.

Reply:
Good to know the option is there, but feels like that might be an improvement for a different PR.